client/js/components/map/admin/AdminLayerList.vue

```diff
@@ -197,28 +197,28 @@
             {{ Translator.trans('map.base.minimap.hint') }}
           </p>
         </div>
-        <div
-          class="text-right u-mv space-inline-s"
-          v-if="false === isLoading">
-          <dp-button
-            data-cy="adminLayerList:save"
-            :busy="!isEditable"
-            :text="Translator.trans('save')"
-            @click="saveOrder" />
-          <dp-button
-            data-cy="adminLayerList:saveAndReturn"
-            :busy="!isEditable"
-            :text="Translator.trans('save.and.return.to.list')"
-            @click="saveOrder(true)" />
-          <button
-            class="btn btn--secondary"
-            data-cy="adminLayerList:resetOrder"
-            type="reset"
-            @click.prevent="resetOrder">
-            {{ Translator.trans('reset.order') }}
-          </button>
-        </div>
       </template>
+      <div
+        class="text-right u-mv space-inline-s"
+        v-if="false === isLoading">
```

Comment:
```suggestion
        v-if="!isLoading">
```

I think there are several comparisons that could be simplified in this file.